# QA warnings
# * ../win/tty/wintty.c:2830:27: warning: implicit declaration of function ‘open’; did you mean ‘popen’? [-Wimplicit-function-declaration] 
Signed-off-by: Nic Boet <nic@boet.cc>

--- a/win/tty/wintty.c      2021-09-18 22:39:42.711000000 -0500
+++ b/win/tty/wintty.c    2021-09-18 22:39:52.948000000 -0500
@@ -2827,7 +2827,7 @@
 #ifdef DEF_PAGER /* this implies that UNIX is defined */
     {   
         /* use external pager; this may give security problems */
-        register int fd = open(fname, 0);
+        register int fd = popen(fname, 0);

         if (fd < 0) {
             if (complain)

